### PR TITLE
status: 2023q1: cloud-init: suggestions

### PR DESCRIPTION
**Pull request by grahamperrin**

### CODE REVIEW

File: website/content/en/status/report-2023-01-2023-03/cloud-init.adoc [hunk 1]
```diff
@@ -12,7 +12,7 @@ Unfortunately, cloud-init support for operating systems other than Linux is rath
 To remedy the situation, this project aims to bring FreeBSD cloud-init support on par with Linux support.
 The broader plan is to lift support across all BSDs.
 
-This quarter has been going very, very slowly, for personal reasons — but also for lack of access to the right resources.
```
Comment (grahamperrin) [inline website/content/en/status/report-2023-01-2023-03/cloud-init.adoc:15]: You might say: 

> … not only for personal reasons but also …

Better, I think: 

> … for personal reasons — also …

**Review by grahamperrin — COMMENTED** [review]
@igalic

**Review by lsalvadore — APPROVED** [review]
Approved by status.

File: website/content/en/status/report-2023-01-2023-03/cloud-init.adoc [hunk 2]
```diff
@@ -12,7 +12,7 @@ Unfortunately, cloud-init support for operating systems other than Linux is rath
...
+This quarter has been going very, very slowly, for personal reasons — also for lack of access to the right resources.
```
Comment (igalic) [inline website/content/en/status/report-2023-01-2023-03/cloud-init.adoc:15]: also, use -- which AsciiDoctor automatically converts to em-dash for us
```suggestion
This quarter has been going very, very slowly, for personal reasons -- also for lack of access to the right resources.
```

**Review by igalic — COMMENTED** [review]
good! and also:

**Review by igalic — APPROVED** [review]
👍

**Pull request closed without merging**